### ISSUE TITLE
Authorship displays only the user's GitHub profile image but no direct means to identify

### ISSUE THREAD

**Issue by JakobJingleheimer** [nodejs/nodejs.org#7093]
### Enter your suggestions in details:

Currently, the docs include authorship by means of the contributors' GitHub profile image. However, it does not cite the username or link to the profile. For a sighted user, this is very limited. This also creates an accessibility failure: authorship is effectively unavailable to a blind user.

A simple `title` attribute of the author's GitHub handle (or name) would be a good start. This could also get fancier beyond the browser's default display of `title` via a tooltip. On top of that, the image could link to the user's profile.

**Comment by AugustinMauroy** [comment]
Part of the problem is that fall back and alt are the same thing at the moment.
The alt should be the username and the fallback should be the acronym.

Also, I thinks we should add in second time a Tooltips (hover behavior) on avatar with GitHub icon + GitHub username.

**Comment by AugustinMauroy** [comment]
> Also, I thinks we should add in second time a Tooltips (hover behavior) on avatar with GitHub icon + GitHub username.

We can re-open this pr if we want that
